Doc/library/string.rst

```diff
@@ -231,8 +231,11 @@ attribute using :func:`getattr`, while an expression of the form ``'[index]'``
 does an index lookup using :func:`__getitem__`.
 
 .. versionchanged:: 3.1
-   The positional argument specifiers can be omitted, so ``'{} {}'`` is
-   equivalent to ``'{0} {1}'``.
+   The positional argument specifiers can be omitted, for :meth:`str.format`,
+   so ``'{} {}'`` is equivalent to ``'{0} {1}'``.
```

Comment:
Since you're now referencing both `str.format()` and `Formatter` in close proximity, I'd change this example to actually reference `.format()` . So something like: `'"{} {}'.format(a, b) is equivalent to '{0} {1}'.format(a, b)"`.